hi-ml-cpath/src/health_cpath/preprocessing/create_panda_tiles_dataset.py

```diff
@@ -304,12 +256,12 @@ def main(panda_dir: Union[str, Path], root_output_dir: Union[str, Path], level:
     parser.add_argument(
         "--overwrite",
         action="store_true",
-        default=False,
+        default=True,
     )
     parser.add_argument(
         "--filter-slide",
         type=str,
-        default="",
+        default="b896",  # filtering for "b896" gives 4 slides, good for debugging
```

Comment:
[Minor] Should this ID prefix maybe be mentioned in the help, rather than set as default? Otherwise, it's awkward to have to add `--filter-slide=""` to run the script on the full dataset.